experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -107,3 +112,8 @@ Occurs: 1 times
 Redacted compiler error message:
 unmatched actual "REDACTED" in call
 Raw compiler error message:
+--
+Occurs: 1 times
++===========================GNAT BUG DETECTED==============================+
+| GNU Ada (ada2goto) Assert_Failure failed precondition from range_check.ads:29|
```

Comment:
Yeah... I realise this is out of scope for this PR but can we fix ASAP.  We shouldn't have compile errors in the golden results.

Reply:
We already have a PR for this: https://github.com/diffblue/gnat2goto/pull/225.